packages/web-components/src/split-button/split-button.stories.ts

```diff
@@ -0,0 +1,225 @@
+    },
+    formTarget: {
+      control: 'text',
+      description: 'The target frame or window to open the form submission in.',
+      name: 'formtarget',
+      table: { category: 'attributes', type: { summary: 'string' } },
+    },
+    iconOnly: {
+      control: 'boolean',
+      description: 'Indicates the button should only display as an icon.',
+      name: 'icon-only',
+      table: { category: 'attributes', type: { summary: 'boolean' } },
+    },
+    name: {
+      control: 'text',
+      description:
+        "The name of the element. This element's value will be surfaced during form submission under the provided name.",
+      table: { category: 'attributes', type: { summary: 'string' } },
+    },
+    size: {
+      control: 'select',
+      description: 'The size of the button.',
+      options: ['', ...Object.values(ButtonSize)],
+      mapping: { '': null, ...ButtonSize },
+      table: {
+        category: 'attributes',
+        type: { summary: Object.values(ButtonSize).join('|') },
+      },
+    },
+    shape: {
+      control: 'select',
+      description: 'The shape of the button.',
+      options: ['', ...Object.values(ButtonShape)],
+      mapping: { '': null, ...ButtonShape },
+      table: {
+        category: 'attributes',
+        type: { summary: Object.values(ButtonShape).join('|') },
+      },
+    },
+    type: {
+      control: 'select',
+      description: 'The type of the button.',
+      options: ['', ...Object.values(ButtonType)],
+      mapping: { '': null, ...ButtonType },
+      table: {
+        category: 'attributes',
+        type: { summary: Object.values(ButtonType).join('|') },
+      },
+    },
+    value: {
+      control: 'text',
+      description: 'The initial value of the button.',
+      table: { category: 'attributes', type: { summary: 'string' } },
+    },
+    slottedContent: {
+      control: false,
+      description: 'The default slot',
+      name: '',
+      table: { category: 'slots', type: {} },
+    },
+    startSlottedContent: {
+      control: false,
+      description: 'Slot for start icons',
+      name: 'start',
+      table: { category: 'slots', type: {} },
+    },
+    endSlottedContent: {
+      control: false,
+      description: 'Slot for end icons',
+      name: 'end',
+      table: { category: 'slots', type: {} },
+    },
+  },
+} as Meta<FluentSplitButton>;
+
+export const Default: Story = {};
+
+export const Appearance: Story = {
+  render: renderComponent(html`<div>${repeat(story => story.storyItems, storyTemplate)}</div> `),
+  args: {
+    storyItems: [
+      { slottedContent: () => 'Default' },
+      { slottedContent: () => 'Primary', appearance: ButtonAppearance.primary },
+      { slottedContent: () => 'Outline', appearance: ButtonAppearance.outline },
+      { slottedContent: () => 'Subtle', appearance: ButtonAppearance.subtle },
+      { slottedContent: () => 'Transparent', appearance: ButtonAppearance.transparent },
+    ],
+  },
+};
```

Comment:
It's interesting we apply `appearance|size|shape` at the Story level in storybook like it's a prop of the split button component, but split-button has no props. Do we need additional docs to instruct authors to apply presentation attributes on both buttons? I think we have a good approach but want it to be obvious to partners.